RELEASE_NOTES.md

```diff
@@ -1,1326 +1,284 @@
+* CASMCMS-9126 Console - log permissions get set incorrectly
+* CASMHMS-6294 SMD: Investigate Scaling Issues in CSM 1.5
+* CASMHMS-6325 vShasta: HSM and PCS tests fail after 1.4 > 1.5 upgrade
+* CASMTRIAGE-7594 cray-console pods keep disconnecting conman sessions.
+* CASMPET-7291 Review csm-rie:1.4.0 (142 days)
+* CRAYSAT-1945 Bug: For lesser page size, cfs v2 session throws traceback error
+* CASMTRIAGE-7346 Upgrade of ncn-m001 to csm-1.6.0-beta.1 is failing setting NTP
+* CRAYSAT-1875 Add new HSM types to sat status
+* CASMTRIAGE-7607 vShasta: upgrade 1.5 > 1.6: cray-nexus deployment fails in prerequisites.sh
+* CASMNET-2270 Exclude cray-shared-kafka-entity-operator network policy during Cilium live migration
+* CASMCMS-9068 Allow customization of ipxe debug options
+* CASMCMS-9236 Fix BOS migration bug in CSM 1.6.1
+* CASMTRIAGE-7559 Lemondrop: CFS layer fails when upgraded to 25.3
+* CASMHMS-6277 FAS: Investigate security fix from Dependabot
+* CASMSEC-505 Kyverno background policy scans are ignoring resourceFilters
+* STP-3724 Finalize docs-sat move to docs-csm
+* CASMINST-7108 Simplify license checker filename pattern override
+* CASMHMS-6239 PCS: ETCD requests are too large at scale
+* CASMTRIAGE-7567 Observed several thousand restarts of cray-sysmgmt-health-redfish-exporter on fanta
+* CRAYSAT-1974 Resolve dependabot alerts (Jinja2)
+* CASMTRIAGE-7627 check if cray-spire jwks and velero backup tests need additional logic
+* CASMHMS-6324 Set up and run 'pprof' against HMS services to find memory leaks
+* CASMTRIAGE-7663 Compute node CFS configuration failing with key issue
+* CASMCMS-9245 Limit requests_retry_session version
+* CASMTRIAGE-7682 Tyr:  March product set - ARM image fails to customize with CFS.
+* CASMINST-3816 manually copying large files into s3fs cache directory prevents prune from pruning them
+* CASMINST-7114 TESTS: rgw_endpoint_check throwing python error
+* CASMCMS-9255 BOS: Image Regular Expression Fragile
+* CASMCMS-9241 cfs-debugger: 'NoneType' object has no attribute 'group'
+* CASMCMS-9201  IMS artifacts remained orphaned with CSM 1.5.2 systems
+* CASMTRIAGE-7715 log files permissions changed manually remain unchanged
+* CASMMON-469 delete SMa postgres VMscrapeserive  for SMA
+* CASMMON-475 seeing errors in the log systmgmt-health-redfish-exporter after configuring E100-smart-data
+* CASMTRIAGE-7823 Install Pipeline -  management-nodes-rollout failed with 503
 
 ## Deprecations
 
-* `sat swap cable` and `sat swap switch` commands
-* [Data Virtualization Service (DVS)](glossary.md#data-virtualization-service-dvs)
+*
 
 For a list of all deprecated CSM features, see [Deprecations](introduction/deprecated_features/README.md#deprecations).
 
 ## Removals
 
-* [BOS](glossary.md#boot-orchestration-service-bos) v1
-* User Access Service
-* User Access Instance
+* CASMCMS-9035 Remove sshd from cray-console-operator image
```

Comment:
I don't think this is a removal at the CSM level. This would make more sense under the security improvements section
```suggestion
```

Reply:
Moving.

Reply:
Removing - it already exists in the sec improv section.